cylc/uiserver/main.py

```diff
@@ -42,13 +42,13 @@
 
 
 class MyApplication(web.Application):
-    is_closing = False
+    is_closing: bool = False
 
-    def signal_handler(self, signum, frame):
+    def signal_handler(self, signum: Any, frame: Any) -> None:
         logger.info('exiting...')
         self.is_closing = True
 
-    def try_exit(self, uis):
+    def try_exit(self, uis: "CylcUIServer") -> None:
```

Comment:
Need to use a string to define the type because `CylcUIServer` is defined further down in this file, and `CylcUIServer` also uses this type (cyclic) so one would need to use this trick of string type.